packages/graphql/src/resolvers/marketplace/_featuredAndHidden.js

```diff
@@ -21,12 +21,12 @@ export async function getFeatured(net) {
   if (net === 'rinkeby') netId = 4
   if (!netId) return []
 
+  if (featured[netId]) return featured[netId]
+
   queues.featured = queues.featured || new Queue()
   if (queues.featured.fetching) await queues.featured.isDone()
   queues.featured.fetching = true
 
-  if (featured[netId]) return featured[netId]
```

Comment:
Don't we actually need to keep this and put it in right after line 25 ?

Reply:
Ah yea you're right